TestAssets/TestProjects/PerformanceTestProjects/TwoTargetApp/project.json

```diff
@@ -3,14 +3,31 @@
   "buildOptions": {
     "emitEntryPoint": true
   },
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "type": "platform",
-      "version": "1.0.0-rc3-004324"
+  "dependencies": { },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-004336"
+        }
+      }
+    },
+    "netstandard1.6": {
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc3-24127-01",
```

Comment:
That doesn't work. Microsoft.NETCore.App doesn't support any 'netstandard' TFM.

See https://github.com/dotnet/core-setup/pull/24

/cc @weshaggard